1080i/Includes_Variables.xml

```diff
@@ -169,6 +169,7 @@
     <value condition="Skin.HasSetting(MusicEntry.Playlists)">ActivateWindow(Music,Playlists,return)</value>
     <value condition="Skin.HasSetting(MusicEntry.Files)">ActivateWindow(Music,Files,return)</value>
     <value condition="Skin.HasSetting(MusicEntry.Albums)">ActivateWindow(Music,Albums,return)</value>
+    <value condition="Skin.HasSetting(MusicEntry.Genres)">ActivateWindow(Music,Genres,return)</value>
     <value>ActivateWindow(Music,Artists,return)</value>
   </variable>
```

Comment:
You also need to add a value for MusicEntryPointLabel2 to show the selected setting within the settings menu